.circleci/config.yml

```diff
@@ -65,33 +85,88 @@ jobs:
             - save-fhirbridge-test-results
 
 
-    # build-and-test-fhirbridge:
-    #     description: |
-    #         Build and test FHIR-bridge, analyze code, collect code coverage metrics,
-    #         upload results to sonarcloud.io
-    #     executor: docker-py3-java11-postgres
-    #     steps:
-    #         - checkout
-    #         - install-maven
-    #         - git-clone-ehrbase-repo
-    #         - cache-out-fhirbridge-m2-dependencies
-    #         - cache-out-ehrbase-m2-dependencies
-    #         - build-ehrbase
-    #         - start-ehrbase-server
-    #         - build-and-test-fhirbridge
-    #         - cache-in-ehrbase-m2-dependencies
-    #         - cache-in-fhirbridge-m2-dependencies
-    #         - start-fhirbridge-application
-    #         - run-robot-integration-tests
-    #         - save-fhirbridge-robot-test-results
-    #         # - collect-fhirbridge-unittest-results
-    #         - collect-fhirbridge-integrationtest-results
-    #         - save-fhirbridge-test-results
-    #         - save-jacoco-coverage-report
-    #         - sonarcloud/scan:
-    #             cache_version: 1
+    build-and-test-fhirbridge:
+        description: |
+            Build and test FHIR-bridge, analyze code, collect code coverage metrics,
+            upload results to sonarcloud.io
+        executor: docker-py3-java11-postgres
+        steps:
+            - checkout
+            - install-maven
+            - git-clone-ehrbase-repo
+            - cache-out-fhirbridge-m2-dependencies
+            - cache-out-ehrbase-m2-dependencies
+            - build-ehrbase
+            - start-ehrbase-server
+            - build-and-test-fhirbridge
+            - cache-in-ehrbase-m2-dependencies
+            - cache-in-fhirbridge-m2-dependencies
+            - start-fhirbridge-application
+            - run-robot-integration-tests
+            - save-fhirbridge-robot-test-results
+            # - collect-fhirbridge-unittest-results
+            - collect-fhirbridge-integrationtest-results
+            - save-fhirbridge-test-results
+            - save-jacoco-coverage-report
+            - sonarcloud/scan:
+                cache_version: 1
 
 
+    tag-version:
+        docker:
+            - image: circleci/openjdk:stretch
+        steps:
+            - checkout
+            - configure-git-for-ci-bot
+            - run:
+                  name: Update Snapshot Version
+                  description: |
+                      This automatically sets the application version when there is a commit to release or hotfix branch.
+                      The version number is parsed from the branch name, for example release/v1.0.5 will create a tag v1.0.5
+                      and set the version number in pom.xml to same version.
+                  command: |
+                      VERSION_NUMBER="$(echo $CIRCLE_BRANCH | awk -F'/v' '{print $2;}')"
```

Comment:
I would add this to the num-portal as well, in order to be consistent.

Reply:
👍  OK